### ISSUE TITLE
Keyboard focus is not sequential order in the 'Left navigation' in the 'SQL Pref Main' page, while navigating using 'Tab' key.

### ISSUE THREAD

**Issue by JamesFerebee** [microsoft/SqlNexus#339]
Impact on a Person with Disability (PWD):
Users who rely on keyboard are getting confused and losing track of focus as Keyboard focus is not sequential order in the 'Left navigation' in the 'SQL Pref Main' page, while navigating using 'Tab' key.

Test Environment:​
OS: Windows 11 [Version 22H2 (OS Build 22621.2283)].

Desktop application: SQL Nexus.

Pre-Requisites:
1. Install SQL NEXUS application on the desktop.

Repro Steps:
Step 1: Open the SQL Nexus app in desktop.
Step 2: Connect to server Dialog gets displayed. Connect to the server by invoking connect button.
Step 3: 'SQL Pref Main' page getting displayed-Navigate the left navigation section using key board tab key.
Step 4: Verify whether Keyboard focus is on sequential order in the 'Left navigation' in the 'SQL Pref Main' page, while navigating using 'Tab' key or not.
 

Actual Result:
Keyboard focus is not sequential order in the 'Left navigation' in the 'SQL Pref Main' page, while navigating using 'Tab' key.
Current focus order: Report->SQL Pref Main->Power Bi report->Logs->Open nexus log->Open Readrace log->Tasks->print->Import->Export->Copy to clipboard->Edit Custom rowset.

Note: Same issue reproes with screen readers as well.

Expected Result:

Keyboard focus should be sequential order 'Left navigation' in the 'SQL Pref Main' page, while navigating using 'Tab' key.
Expected focus order:  Report->SQL Pref Main->Power Bi report->Tasks->Import->Edit Custom rowset->Export->print->Copy to clipboard.

Suggestive Fixes:
Make sure DOM structure code of all interactive elements present in the ''SQL Pref Main" page, should be in same order like how all interactive elements order is visble in the application.
Ensure that all interactive elements must be in semantic tags.
provide 'tab-index=0' for all interactive elements.

**Comment by JamesFerebee** [comment]
1711612

**Comment by PiJoCoder** [comment]
Fixed by PR https://github.com/microsoft/SqlNexus/pull/365
